src/components/editor/EditorArticleController.jsx

```diff
@@ -349,8 +361,14 @@ export default class EditorArticleController extends FalcorController {
       }
 
       return (
-        <div style={styles.innerPaper}>
-          <h2>Article Editor: {article.title}</h2>
+        <Dialog
+          title="Article Editor"
+          open={this.state.open}
```

Comment:
let's just change this to just `open` (the implicit way of writing `open={true}`) since we never change this, we "close" this by routing away.

Reply:
oh I kinda overlooked this.. will change it now!

Reply:
and thanks for bringing me onboard :D